frontend/src/components/LogoSquare.vue

```diff
@@ -1,8 +1,8 @@
+    <svg class="w-full" fill="currentColor" viewBox="0 0 60 60" xmlns="http://www.w3.org/2000/svg"><path d="M0 55.737v1.324h60v-1.324zM0 53.096v1.322h60v-1.322zM38.436 49.379l-1.185.877a12.523 12.523 0 005.972 1.512l9.108.002-2.852-1.325-6.256-.002a11.238 11.238 0 01-4.787-1.064zM41.881 45.004l-.662 1.191a7.25 7.25 0 002 .282l2.187.002-.77-1.323-1.417-.002a6.143 6.143 0 01-1.338-.15zM42.803 42.479l-.393 1.277a4.644 4.644 0 00.672.073l.983.002-.407-1.325-.437-.002c-.142 0-.281-.008-.418-.025zM43.213 39.867l-.145 1.318c.101.007.186.003.29.002zM56.367 37.215l.08 1.324H60v-1.324zM29.992 37.215a1.995 1.995 0 00-1.984 1.984h1.322c0-.373.289-.662.662-.662.373 0 .662.29.662.662 0 .374-.289.661-.662.664L0 39.865v1.325l29.992-.002a1.999 1.999 0 001.984-1.989 1.995 1.995 0 00-1.984-1.984zM56.049 34.576l.197 1.322H60v-1.322z"/><path d="M29.994 34.576a4.641 4.641 0 00-4.633 4.63h1.326c0-1.833 1.471-3.308 3.305-3.308s3.307 1.475 3.307 3.309-1.472 3.307-3.307 3.303L0 42.506v1.322l29.992.004a4.634 4.634 0 004.631-4.625 4.64 4.64 0 00-4.629-4.631z"/><path d="M29.992 31.926c-4.01 0-7.275 3.266-7.275 7.277h1.322c0-3.296 2.657-5.953 5.953-5.953s5.953 2.657 5.953 5.953-2.658 5.953-5.953 5.953H0v1.323h29.992c4.01 0 7.275-3.265 7.275-7.276 0-4.01-3.265-7.277-7.275-7.277zM55.432 31.924l.346 1.324H60v-1.324zM53.275 26.637l.664 1.32H60v-1.32z"/><path d="M29.992 26.637c-6.933 0-12.568 5.633-12.568 12.566h1.324a11.235 11.235 0 0111.244-11.246 11.235 11.235 0 0111.244 11.246 11.232 11.232 0 01-11.244 11.242H0v1.322l29.992.002c6.933.001 12.568-5.633 12.568-12.566s-5.635-12.566-12.568-12.566z"/><path d="M29.994 23.987c-7.718 0-14.061 5.771-15.041 13.229H0v1.322h15.459l.65-.033.051-.572c.64-7.08 6.582-12.623 13.834-12.623 7.68 0 13.891 6.21 13.891 13.895 0 6.936 5.634 12.568 12.566 12.566H60v-1.322h-3.549a11.232 11.232 0 01-11.246-11.244c0-8.398-6.817-15.217-15.21-15.217z"/><path d="M29.992 18.696c-8.742 0-16.153 5.513-19.1 13.229H-.001v1.324h10.148l1.598-.002.156-.441c2.633-7.448 9.732-12.787 18.09-12.787 10.602 0 19.182 8.581 19.182 19.186 0 4.013 3.266 7.278 7.277 7.277H60V45.16h-3.55a5.94 5.94 0 01-5.952-5.955c0-11.32-9.188-20.508-20.506-20.508z"/><path d="M29.992 16.045c-9.217 0-17.132 5.426-20.855 13.23H0v1.324h9.918l.174-.39c3.426-7.57 11.041-12.84 19.898-12.84 12.063 0 21.828 9.765 21.828 21.832a4.636 4.636 0 004.631 4.63h3.55V42.51h-3.548a3.294 3.294 0 01-3.309-3.308c0-12.782-10.372-23.156-23.15-23.156z"/><path d="M29.992 13.407c-9.666 0-18.056 5.353-22.473 13.229h-7.52v1.322h8.245l.187-.347c4.132-7.67 12.234-12.881 21.561-12.881 13.525 0 24.475 10.95 24.475 24.477 0 1.09.896 1.985 1.985 1.984H60v-1.324h-3.548a.65.65 0 01-.663-.66c0-14.24-11.557-25.8-25.797-25.8z"/></svg>
 </template>
 <script>
 export default {
-    name: "LogoSquare"
+    name: 'FlowForgeLogo'
```

Comment:
This component is no longer used and can be removed. Alternatively it needs a different name as this one is already used by `Logo.vue`.